modules/ossm-control-plane-templates.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/prepare-to-deploy-applications-ossm.adoc
+
+[id="ossm-control-plane-templates_{context}"]
+= Creating control plane templates
+
+When you configure control plane templates, which follow the same syntax as the `ServiceMeshControlPlane`, users inherit settings in a hierarchical factor. The Operator is delivered with a `default` template with default settings for {ProductName}. To add custom templates you must create a ConfigMap named `smcp-templates` in the `openshift-operators` and mount the ConfigMap in the Operator at `/usr/local/share/istio-operator/templates`.
+
+[id="ossm-create-configmap_{context}"]
+== Creating the ConfigMap
+
+.Prerequisites
+
+* An installed, verified {ProductShortName} Operator.
+* An account with cluster administrator access.
+* Location of the Operator deployment.
+
+.Procedure
+
+. Open the {product-title} web console in a Web browser.
+
+. Log into your {product-title} installation as a cluster administrator.
+
+. Run this command to create the ConfigMap and substitute `templates-directory` with the location of the `ServiceMeshControlPlane` files:
++
+----
+ oc create configmap --from-file=<templates-directory> smcp-templates -n openshift-operators
+----
+
+. Locate the Operator deployment.
++
+----
+oc get clusterserviceversion -n openshift-operators | grep 'Service Mesh'
+maistra.v1.0.0            Red Hat OpenShift Service Mesh   1.0.0                Succeeded
+----
+
+. Edit the Operator cluster service version to instruct the Operator to use the `scmp-templates` ConfigMap.
```

Comment:
`scmp-templates` should be `smcp-templates`

Reply:
Fixed.